.env.example

```diff
@@ -26,4 +26,11 @@ DISCORD_CHANNEL_UPDATE=
 
 # The website address used to access this epochtal instance. Please make sure this URL matches the external URL of your epochtal deployment.
 # Excluding protocol ("https://"), but including port if non-standard. For example: "epochtal.p2r3.com" or "epochtal.example.com:20000"
-#WEB_URL=
+# If left unset, defaults to "localhost:8080"
+WEB_URL=
+# Whether to host the webserver with TLS (requires a certificate)
+# If left unset, defaults to "false"
+USE_TLS=
+# Whether clients are expected to connect to the "https://" version of the site - usually the same as USE_TLS unless using a proxy
+# If left unset, defaults to "false"
+USE_HTTPS=
```

Comment:
In my opinion, the envvars themselves should also be commented out, to make it slightly more obvious that they are optional.

If you don't wanna do that, at least remove `Optional environment variables are commented out.` on line 2 lmao